wagtail/admin/templates/wagtailadmin/pages/page_listing_header.html

```diff
@@ -7,11 +7,9 @@ <h1 class="w-sr-only">
         {{ title }}
     </h1>
     {# breadcrumbs #}
-    {% if parent_page.is_root %}
-        <div class="w-pl-3">{% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' is_expanded=True %}</div>
-    {% else %}
-        {% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' %}
-    {% endif %}
+    <div class="w-py-3{% if parent_page.is_root %} w-pl-3{% endif %}">
+        {% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' is_expanded=parent_page.is_root %}
+    </div>
     {# Actions divider #}
```

Comment:
Change looks good - I'll do a test locally and get this merged in later today

Well done - a nice simple solution here.